package.json

```diff
@@ -12,6 +12,8 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.4.3",
     "react-scripts": "^5.0.1",
+    "redux": "^4.2.0",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
- [ ] Your Linters are not working 😨 Kindly fix your linters because linters help you not introduce those errors which can break your application 👍